foundations/foundation-config/src/test/java/io/servicecomb/config/TestConfigurationSpringInitializer.java

```diff
@@ -32,4 +36,11 @@ public void testAll() {
         Assert.assertEquals(3, listO.size());
         Assert.assertEquals(null, ConfigUtil.getProperty("notExist"));
     }
+
+    @AfterClass
+    public static void tearDown() throws Exception {
```

Comment:
infact, i feel that move ArchaiusUtils to src/main/java of foundation-config is not so bad
but seanyinx has different idea